Readme.md

```diff
@@ -240,55 +183,55 @@ final class SomethingHappenedElsewhere implements EventInterface, ProvidesEventT
 }
 ```
 
-### Generic Fault (WIP)
-
-* [x] **GenericFault**: event triggered by the EventBus is an event handler throw an exception
-
-### How to implement you own event listener ?
-
-Your must implement the ```EventListenerInterface```:
+### CommandHandlers
 
 ```php
-class ConsoleOutputListener implements EventListenerInterface
+/**
+ * @Flow\Scope("singleton")
+ */
+final class ButtonCommandHandler
 {
     /**
-     * @var SystemLoggerInterface
+     * @var ButtonRepository
      * @Flow\Inject
      */
-    protected $systemLogger;
+    protected $buttons;
 
     /**
-     * @param ButtonTagged $event
-     * @param EventMetadata $metadata
+     * @param CreateButton $command
      */
-    public function onButtonTagged(ButtonTagged $event, EventMetadata $metadata)
+    public function handleCreateButton(CreateButton $command)
     {
-        $this->systemLogger->log('--- ConsoleOutputListener say something has been tagged ---');
+        $button = new Button($command->getIdentifier());
+        $button->changeLabel($command->getLabel());
+
+        $this->buttons->save($button);
```

Comment:
Mh.. `changeLabel` is a different intention..
And `save()` is not the right method here, is it?
While we're on it we should rewise that a little IMO, for example:

```php
public function handleInitializeButton(InitializeButton $command)
{
    $button = Button::initialize($command->getButtonIdentifier(), $command->getLabel());
    $this->buttons->add($button);
}
```